src/site/content/en/fast/codelab-text-compression/index.md

```diff
@@ -29,17 +29,15 @@ the app's request size.
 Before diving in to add optimizations, it's always a good idea to first analyze
 the current state of the application.
 
-+ To preview the site, mouse over the editor, press the **App** button, then the
-  **Show** button.
+{% Instruction 'preview' %}
 
 This app, which was also covered in the ["Remove unused
 code"](/remove-unused-code) codelab, lets you vote for your favorite
 kitten. 🐈
 
 Now take a look at how large this application is:
 
-+  Open the DevTools by pressing `Command+Option+i / Ctrl+Shift+i`.
-+  Click on the **Network** panel.
+{% Instruction 'devtools-network' %}
 
 <img class="w-screenshot" src="./network-tab.png" alt="Network panel">
```

Comment:
Is this needed? We don't typically provide it, and I'm not sure it's adding much with so little of the screen shown.

Also the subsequent line doesn't follow Documentation Style Guide. Should be: "Select the Disable cache checkbox and reload the app."

Reply:
Nope we can lose it.

Reply:
Fixed in new PR